### ISSUE TITLE
test_queries_with_read_repair seems flaky

### ISSUE THREAD

**Issue by xemul** [scylladb/scylladb#20217]
While validating unrelated change (#20172) the test failed with

```
            if i % 1000 == 0:
                ck = random.randint(i - 1000, i)
                session.execute(delete_stmt, (0, ck))
                node2.flush()
>               expected_rows.remove({"ck": ck, "v": 2 * ck})
E               ValueError: list.remove(x): x not in list

```

https://jenkins.scylladb.com/job/scylla-master/job/scylla-ci/10936/testReport/junit/reversed_queries_test/TestReversedQueriesReadRepairBigMutationsSmallChanges/Tests___dtest_with_topology_changes___test_queries_with_read_repair/

**Comment by paszkow** [comment]
Already fixed by https://github.com/scylladb/scylla-dtest/pull/4807.

**Comment by denesb** [comment]
> Already fixed by [scylladb/scylla-dtest#4807](https://github.com/scylladb/scylla-dtest/pull/4807).

Why is this issue not closed if the problem is fixed? Anything else need to be done here?
